src/logic/import/voc/VOCImporter.ts

```diff
@@ -0,0 +1,98 @@
+import {ImageData, LabelName, LabelRect} from '../../../store/labels/types';
+import {LabelUtil} from "../../../utils/LabelUtil";
+import {AnnotationImporter} from '../AnnotationImporter';
+import {LabelsSelector} from '../../../store/selectors/LabelsSelector';
+
+export type FileParseResult = {
+    filename: string,
+    labeledBoxes: LabelRect[]
+};
+export type VOCImportResult = {
```

Comment:
I remember why I had this around now, it was because I was trying out the reduce version. From comment below, I implemented it that way so going to leave this type in but not export